### PR TITLE
Clear search button

### PR DESCRIPTION
**Pull request by Allegraknox**
Added clear search button. Implements mariam's button class so it matches search button. Simply sends user back to the view that is shown when a search of "" is entered.

### CODE REVIEW

File: app/controllers/items_controller.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ class ItemsController < ApplicationController
   def has_user
     unless current_user
       flash[:warning] = 'You must be logged in to perform this action.'
-      redirect_to new_user_path
+      redirect_to items_path
```
Comment (amycsong98) [inline app/controllers/items_controller.rb:8]: why was this changed?

File: app/controllers/items_controller.rb [hunk 2]
```diff
@@ -68,14 +69,14 @@ def create
   end
   
   def search
-    if !params[:search].empty?
+    if params[:search] == nil || params[:search].empty?
```
Comment (amycsong98) [inline app/controllers/items_controller.rb:72]: if `params[:search].nil?`

**Review by amycsong98 — APPROVED** [review]
LGTM

File: app/controllers/items_controller.rb (hunk 1 above)
Comment (amycsong98) [reply app/controllers/items_controller.rb:8]: i don't think this should be changed

Comment (Allegraknox) [reply app/controllers/items_controller.rb:8]: New user path doesnt exist anymore @amycsong98 , since we can't create a user its just sign in with google SSO

File: app/controllers/items_controller.rb (hunk 2 above)
Comment (Allegraknox) [reply app/controllers/items_controller.rb]: changed button_to to link_to so the clear search functionality doesn't touch the controller anymore (just links to root_path)

**Review by njackson1-code — APPROVED** [review]
Looks good!

File: app/controllers/items_controller.rb (hunk 1 above)
Comment (amycsong98) [reply app/controllers/items_controller.rb:8]: cool
